.rhcicd/grafana/grafana-dashboard-subscription-watch.configmap.yaml

```diff
@@ -43,7 +43,7 @@ data:
             {
               "datasource": {
                 "type": "prometheus",
-                "uid": "${datasource}"
+                "uid": "${aws_kafka_datasource}"
```

Comment:
My reasoning to name this datasource as "aws_kafka_datasource" is that the metrics start with "aws_kafka_*".
Feel free to suggest other more appropiate name.

Reply:
Makes sense. I am fine with this name